include/kernels/color_spinor_pack.cuh

```diff
@@ -237,6 +269,9 @@ namespace quda {
           arg.field.Ghost(dim, dir, spinor_parity, ghost_idx, s, c, 0, max) = arg.field(spinor_parity, x_cb, s, c);
         }
       }
+#ifdef NVSHMEM_COMMS
+      if (arg.shmem & 1) shmem_signalwait(0, 0, (arg.shmem & 4), arg);
```

Comment:
Why not just `if (arg.shmem)`? (simpler SASS)

Reply:
I'll fix that